### PR TITLE
AV-190240 : Updating helm related installation docs for AKO 1.11.1

### PR DESCRIPTION
**Pull request by DixitAakash**

### CODE REVIEW

**Comment by DixitAakash** [comment]
build ako

**Comment by DixitAakash** [comment]
build ako

File: docs/install/helm.md [hunk 1]
```diff
@@ -105,23 +105,23 @@ kubectl apply -f <output_dir>/ako/crds/
...
+helm show values oci://projects.registry.vmware.com/ako/helm-charts/ako --version 1.11.1 > values.yaml
 
 ```
 *Step4*
 
 Upgrade the helm chart
 
 ```
-helm upgrade ako-1593523840  oci://projects.registry.vmware.com/ako/helm-charts/ako -f /path/to/values.yaml --version 1.10.3 --set ControllerSettings.controllerHost=<IP or Hostname> --set avicredentials.password=<username> --set avicredentials.username=<username> --namespace=avi-system
+helm upgrade ako-1593523840  oci://projects.registry.vmware.com/ako/helm-charts/ako -f /path/to/values.yaml --version 1.11.1 --set ControllerSettings.controllerHost=<IP or Hostname> --set avicredentials.password=<username> --set avicredentials.username=<username> --namespace=avi-system
```
Comment (akshayhavile) [inline docs/install/helm.md:124]: @DixitAakash : There is helm upgrade command present in ako_ha.md. Please update that one also.

File: docs/install/helm.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Step 4: Check the installation
 helm list -n avi-system
 
 NAME          	NAMESPACE 	REVISION	UPDATED     STATUS  	CHART    	APP VERSION
-ako-1691752136	avi-system	1       	2023-08-11	deployed	ako-1.10.3	1.10.3
+ako-1691752136	avi-system	1       	2023-08-11	deployed	ako-1.11.1	1.11.1
```
[the remark this answers is not in the document]
Comment (DixitAakash) [reply docs/install/helm.md]: done

File: docs/install/helm.md (hunk 1 above)
Comment (DixitAakash) [reply docs/install/helm.md:124]: @akshayhavile done

**Review by akshayhavile — APPROVED** [review]

File: docs/openshift/openshift_helm.md [hunk 3]
```diff
@@ -55,7 +55,7 @@ Verify the installation
 helm list -n avi-system
 
 NAME          	NAMESPACE 	REVISION	UPDATED     STATUS  	CHART    	APP VERSION
-ako-1691752136	avi-system	1       	2023-08-11	deployed	ako-1.10.3	1.10.3
+ako-1691752136	avi-system	1       	2023-08-11	deployed	ako-1.11.1	1.11.1
```
Comment (pkoshtavmware) [inline docs/openshift/openshift_helm.md:58]: Might want to change date here as well

**Review by pkoshtavmware — DISMISSED** [review]
LGTM

**Comment by DixitAakash** [comment]
build ako

File: docs/openshift/openshift_helm.md (hunk 3 above)
Comment (DixitAakash) [reply docs/openshift/openshift_helm.md]: done

**Review by pkoshtavmware — APPROVED** [review]
LGTM

**Review by akshayhavile — APPROVED** [review]

**Pull request merged**